src/quantuminspire/api.py

```diff
@@ -812,8 +812,8 @@ def _generate_error_result(message: str) -> Dict[str, Any]:
         return result_obj
 
     def execute_qasm(self, qasm: str, backend_type: Optional[Union[Dict[str, Any], int, str]] = None,
-                     number_of_shots: int = 256, collect_tries: Optional[int] = None,
-                     default_number_of_shots: int = 256, identifier: Optional[str] = None,
+                     number_of_shots: int = 1024, collect_tries: Optional[int] = None,
```

Comment:
Shouldn't this be None, with a check that if `number_of_shots` is None, it should be retrieved from the backend_type object?

Reply:
You are right. Nice suggestion